go/vt/vtgate/plan_executor_test.go

```diff
@@ -1089,8 +1088,8 @@ func TestPlanExecutorOtherRead(t *testing.T) {
 
 	stmts := []string{
 		"analyze table t1",
-		"describe t1",
-		"explain t1",
```

Comment:
these test can be added back.

Reply:
Why? The new queries are also OtherRead statements, so we are still testing the same code paths, right?